app/src/main/AndroidManifest.xml

```diff
@@ -32,21 +32,23 @@
             </intent-filter>
             <intent-filter>
                 <action android:name="android.intent.action.VIEW" />
+
                 <category android:name="android.intent.category.DEFAULT" />
                 <category android:name="android.intent.category.BROWSABLE" />
 
                 <!-- Accepts URIs that begin with "https://chat.susi.ai/app" -->
-                <data android:host="chat.susi.ai" android:pathPrefix="/app" android:scheme="https" />
+                <data
+                    android:host="chat.susi.ai"
+                    android:pathPrefix="/app"
+                    android:scheme="https" />
```

Comment:
Probably, nothing has changed here except for indentation.Let it be in the previous state to avoid spacing

Reply:
It has already been reverted in the last commit